content/how-to/cloud-servers/scheduled-images-faq/index.md

```diff
@@ -1,146 +1,136 @@
+
+The best way to tell is to look at the image metadata, but
+unfortunately, you can't see it in the Cloud Control Panel. By using
+the API, you can see that an image created by the scheduled images service
+has the following metadatum:
 
     org.openstack__1__created_by: scheduled_images_service
 
-For information about using the novaclient tool for scheduled images,
-see [Using python-novaclient to manage scheduled
-images](/support/how-to/using-python-novaclient-to-manage-scheduled-images).
 {{</accordion>}}
 {{<accordion title="Is there a minimum or maximum time between scheduled images?" col="in" href="accordion5">}}
 
-No (see next answer), but daily scheduled images will be roughly 24
+No (see next answer), but daily scheduled images are roughly 24
 hours apart and occur on different dates UTC.
 
 For weekly images, you specify the day of the week (determined by UTC)
-when you'd like your server image created. As some days of the week are
+when you'd like your server image created. Because some days of the week are
 much more popular than others for scheduling images, in rare
-circumstances we may create your weekly scheduled image in a window
+circumstances we might create your weekly scheduled image in a window
 beginning 12:00 UTC the day before the day of the week you specify and
 ending at 12:00 UTC the day after the day of the week you specify.
 {{</accordion>}}
 {{<accordion title="Will they occur on the same time each day?" col="in" href="accordion6">}}
 
-In order to operate the scheduled images service with minimal impact on
-on-demand snapshots and other network-intensive data transfers, the time
-a snapshot occurs may be changed at any time to optimize load.  We
-reserve the right to modify the time your image is made so that we can
-balance the number of image creations in flight throughout the cloud
+To operate the scheduled images service with minimal impact on
+on-demand snapshots and other network-intensive data transfers, we might
+change the time a snapshot occurs at any time to optimize load.  We
+reserve the right to modify your image's scjheduled time to
+balance the number of image creations in flight across the cloud
 and throughout the day.
 {{</accordion>}}
-{{<accordion title="Will the image created\_at reflect the completion or start time?" col="in" href="accordion7">}}
+{{<accordion title="Does the image created\_at parameter reflect the completion or start time?" col="in" href="accordion7">}}
 
-The implementation uses the normal OpenStack snapshotting process, so
-the created\_at timestamp will be the time the call was made, i.e., at
-the start of the snapshot.  The time it will be available for you to use
-(that is, when its status is ACTIVE) depends on factors such as the size
+The implementation uses the normal OpenStack snapshot process, so
+the created\_at timestamp is when the call occurred at
+the start of the snapshot.  The image's availability time
```

Comment:
Don't make possessive. Use "The availability time of the image..."

Reply:
respectfully disagree. The possessive is more concise.  We don't use it for Companies.  For example, say "The Rackspace policy" instead of "Rackspace's policy".  In this case the possessive is ok.

Reply:
done.